src/main/scala/org/bitcoins/core/gen/ScriptGenerators.scala

```diff
@@ -71,6 +71,21 @@ trait ScriptGenerators extends BitcoinSLogger {
   } yield CSVScriptSignature(csv, sigs, pubKeys)
 
 
+  /** Generator for a contract used within a [[WithdrawScriptSignature]] */
+  def contract: Gen[Seq[Byte]] = for {
```

Comment:
probably need to create a type for 'contract' and create a generator that takes in a `Sha256Hash160Digest`